src/containers/AddProjectDetails/AddProjectDetailsForm.tsx

```diff
@@ -29,13 +40,113 @@ export default function AddProjectDetailsForm() {
+                </Grid>
+            </styledComp.flexRow>
+            <styledComp.flexRow>
+                <Textarea
+                    label={UIConstants.projectActivitiesLastWeek}
+                    {...register("projectActivitiesLastWeek")}
+                    error={!!errors?.projectActivitiesLastWeek}
+                    helperText={
+                        errors.projectActivitiesLastWeek
+                            ? errors?.projectActivitiesLastWeek.message
+                            : null
+                    }
+                />
+            </styledComp.flexRow>
+            <styledComp.flexRow>
+                <Textarea
+                    label={UIConstants.projectActivitiesNextWeek}
+                    {...register("projectActivitiesNextWeek")}
+                    error={!!errors?.projectActivitiesNextWeek}
+                    helperText={
+                        errors.projectActivitiesNextWeek
+                            ? errors?.projectActivitiesNextWeek.message
+                            : null
+                    }
                 />
             </styledComp.flexRow>
+            <styledComp.flexRow>
+                <Textarea
+                    label={UIConstants.projectAccomplishment}
+                    {...register("projectAccomplishment")}
+                    error={!!errors?.projectAccomplishment}
+                    helperText={
+                        errors.projectAccomplishment
+                            ? errors?.projectAccomplishment.message
+                            : null
+                    }
+                />
+            </styledComp.flexRow>
+            <styledComp.flexRow>
+                <Grid item md={6} sm={6} xs={12}>
+                    <Dropdown
+                        label={UIConstants.projectStatus}
+                        {...register("projectStatus")}
+                        error={!!errors?.projectStatus}
+                        onChange={handleChange}
+                        options={projectStatusLabelValues}
+                        helperText={
+                            errors.projectStatus
+                                ? errors?.projectStatus.message
+                                : null
+                        }
+                    />
+                </Grid>
+
+                {/* <Grid item md={6} sm={6} xs={12}>
+                     <Dropdown
+                        label={UIConstants.projectCompletionDate}
+                        {...register("projectCompletionDate")}
+                        error={!!errors?.projectCompletionDate}
+                        onChange={handleChange}
+                        options={channelOwnerLabelValues}
+                        helperText={
+                            errors.projectCompletionDate
+                                ? errors?.projectCompletionDate.message
+                                : null
+                        }
+                    /> 
+                </Grid> */}
```

Comment:
Pls remove the commented out code

Reply:
This structure is partially required, will replace this with the date picker attribute next to the next PR

Reply:
Okay will remove it :)

Reply:
done